File: JAVA LAB/Bubblesort.java
import java.util.Scanner;
class Bubblesort
{
  public static void main(String args[])
  {
    int n,i,j,temp;
    int a[]=new int[20];
    Scanner s=new Scanner(System.in);
    System.out.println("enter number of elements");
    n=s.nextInt();
    System.out.println("enter elements");
    for(i=0;i<n;i++)
    { 
      a[i]=s.nextInt();
    }
    for(i=0;i<n;i++)
    {
      for(j=0;j<n-1;j++)
      { 
        if(a[j]>a[j+1])
        { 
          temp=a[j];
          a[j]=a[j+1];
          a[j+1]=temp;
        }
      }
    }
    System.out.println("the sorted elements are:");
    for(i=0;i<n;i++)
    { 
      System.out.println("\t"+a[i]);
    }
  }
}

File: DS/bubblesort.c
#include<stdio.h>
int main()
{
	int a[10],i,j,temp,n;
	printf("enter size of an array:");
	scanf("%d",&n);
	printf("\nenter the elements:");
	for(i=0;i<n;i++)
	{
		scanf("%d",&a[i]);
	}
	for(i=0;i<n-1;i++)
	{
		for(j=0;j<n-i-1;j++)
		{
			if(a[j]>a[j+1])
			{
				temp=a[j];
				a[j]=a[j+1];
				a[j+1]=temp;
			}
		}
	}
	for(i=0;i<n;i++)
	{
		printf("%d\t",a[i]);
	}
	return 0;
}
